Client/Cliqz/Frontend/Browser/CliqzSearchViewController.swift

```diff
@@ -474,11 +474,12 @@ class CliqzSearchViewController : UIViewController, LoaderListener, WKNavigation
     private func generateTempFileName(_ title: String) -> String {
         var fileName = title
         fileName = fileName.replace(" ", replacement: "_")
-        fileName = fileName.replace("...", replacement: "")
```

Comment:
this function looks like potential attack surface. is it safe to remove `...` and not have `..` replacements?

Reply:
the other option is to use system generated name (TimeStamp) instead of the card title, but in that case when saving it to Files on iOS11 the images will have meaning less names